changes/arm-improvements.tex

```diff
@@ -17,7 +17,7 @@
 contributors have implemented some of the main features from those extensions,
 up to Armv8.3-A.
 
-\subsubsection[Support for the Arm Scalable Vector Extension (SVE)]{Support for the Arm Scalable Vector Extension (SVE)\footnote{by Giacomo Gabrielli, Javier Setoain, and Giacomo Travaglini}}
+\subsubsection[Support for the Arm Scalable Vector Extension (SVE)]{Support for the Arm Scalable Vector Extension (SVE)\footnote{by Gabor Dozsa, Giacomo Gabrielli, Rekai Gonzalez-Alberquilla, Nathanael Premillieu, and Javier Setoain}}
```

Comment:
I was wondering: will these authors be part of the main author list?

Reply:
I would love to add all of these people as authors on the paper! However, as far as I can tell, I never received confirmation from these authors that they want to be authors on the paper. I worry about adding someone as an author without their knowledge and approval.

See https://github.com/darchr/gem5-20-paper#what-i-need-from-you